packages/mdc-slider/README.md

```diff
@@ -0,0 +1,166 @@
+`MDCSlider` emits a `MDCSlider:change` custom event from its root element whenever the slider value
+is changed _and committed_ by way of a user event, e.g. when a user stops dragging the slider or
+changes the value using the arrow keys. The `detail` property of the event is set to the slider
+instance that was affected.
+
+## Usage within Web Frameworks
+
+#### `MDCSliderAdapter`
+
+| Method Signature | Description |
+| --- | --- |
+| `hasClass(className: string) => boolean` | Checks if `className` exists on the root element |
+| `addClass(className: string) => void` | Adds a class `className` to the root element |
+| `removeClass(className: string) => void` | Removes a class `className` from the root element |
+| `getAttribute(name: string) => string?` | Returns the value of the attribute `name` on the root element, or `null` if that attribute is not present on the root element. |
+| `setAttribute(name: string, value: string) => void` | Sets an attribute `name` to the value `value` on the root element. |
+| `removeAttribute(name: string) => void` | Removes an attribute `name` from the root element |
+| `computeBoundingRect() => ClientRect` | Computes and returns the bounding client rect for the root element. Our implementations calls `getBoundingClientRect()` for this. |
+| `registerEventHandler(type: string, handler: EventListener) => void` | Adds an event listener `handler` for event type `type` to the slider's root element |
+| `deregisterEventHandler(type: string, handler: EventListener) => void` | Removes an event listener `handler` for event type `type` from the slider's root element |
+| `registerThumbEventHandler(type: string, handler: EventListener) => void` | Adds an event listener `handler` for event type `type` to the slider's thumb container element |
+| `deregisterThumbEventHandler(type: string, handler: EventListener) => void` | Removes an event listener `handler` for event type `type` from the slider's thumb container element |
+| `registerBodyEventHandler(type: string, handler: EventListener) => void` | Adds an event listener `handler` for event type `type` to the `<body>` element of the slider's document |
+| `deregisterBodyEventHandler(type: string, handler: EventListener) => void` | Removes an event listener `handler` for event type `type` from the `<body>` element of the slider's document |
+| `registerResizeHandler(handler: EventListener) => void` | Adds an event listener `handler` that is called when the component's viewport resizes, e.g. `window.onresize`. |
+| `deregisterResizeHandler(handler: EventListener) => void` | Removes an event listener `handler` that was attached via `registerResizeHandler`. |
+| `notifyInput() => void` | Broadcasts an "input" event notifying clients that the slider's value is currently being changed. The implementation should choose to pass along any relevant information pertaining to this event. In our case we pass along the instance of the component for which the event is triggered for. |
```

Comment:
Replace `"input"` with `MDCSlider:input`, and `"change"` with `MDCSlider:change` below

Reply:
+1